File: Problems/SWEA_4828/main.cpp
#include <iostream>
#include <cstdio>

int main(int argc, char** argv)
{
	if (argc > 1)
		freopen(argv[1], "r", stdin);

	int TC;
	scanf("%d", &TC);
	for (int t = 1; t <= TC; t++)
	{
		long long mn = 1e10, mx = -mn;
		int N; scanf("%d", &N);
		for (int i = 0; i < N; i++)
		{
			int tmp; scanf("%d", &tmp);
			if (mn > tmp) {mn = tmp;};
			if (mx < tmp) {mx = tmp;};
		}
		printf("#%d %lld", t, mx - mn);
	}
}

File: Problems/SWEA_4828/main.py
TC = int(input())

for t in range(1, TC + 1):
    mn = int(1e10)
    mx = -mn
    N = int(input())
    nums = list(map(int, input().split()))
    for tmp in nums:
        if mn > tmp:
            mn = tmp
        if mx < tmp:
            mx = tmp
    print(f"#{t} {mx - mn}")
